src/app/ubs/ubs-admin/services/admin-table.service.ts

```diff
@@ -354,6 +355,27 @@ export class AdminTableService {
     this.filters = this.filters.filter((filteredElem) => !filteredElem[colName]);
   }
 
+  showTooltip(event: any, tooltip: any, font: string, maxLength = 50): void {
```

Comment:
Use a more precise type (MouseEvent) instead of any for better type safety
and ensure event.target is a valid HTMLElement and check the text length
```
const target = event.target as HTMLElement;
const lengthStr = target?.innerText.length;
```